src/mscorlib/model.xml

```diff
@@ -7389,7 +7389,7 @@
     </Type>
     <Type Name="System.Threading.ThreadStartException">
       <Member Status="ImplRoot" Name="#ctor(System.Exception)" /><!-- EE -->
-      <Member Name="#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
+      <Member Status="ImplRoot" Name="#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
```

Comment:
>Status="ImplRoot" [](start = 14, length = 17)

I'm not really sure I understand why this is sometimes required but sometimes it isn't